package.json

```diff
@@ -105,6 +107,7 @@
     "react-router-dom": "^4.3.1",
     "react-timeago": "^4.1.9",
     "web3": "1.0.0-beta.33",
-    "web3-core-promievent": "1.0.0-beta.33"
+    "web3-core-promievent": "1.0.0-beta.33",
+    "web3-utils": "^1.0.0-beta.52"
```

Comment:
is it fine to mix versions like this? (52 and 33)

Reply:
No :-) I'll drop utils package anyway